From 9b91327357ced8124bbdb54da10c72eb8fc3e652 Mon Sep 17 00:00:00 2001
From: Zhipeng Gong <zhipeng.gong@intel.com>
Date: Thu, 26 Jul 2018 10:03:02 +0800
Subject: [PATCH 2089/2367] drm/i915/gvt: enable pv global gtt update by
 default

This patch enables pv ggtt update by default.
It is needed for both uos and sos.

Tracked-On: projectacrn/acrn-hypervisor#994
Signed-off-by: Zhipeng Gong <zhipeng.gong@intel.com>
Reviewed-by: He, Min <min.he@intel.com>
---
 drivers/gpu/drm/i915/i915_params.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/i915/i915_params.c b/drivers/gpu/drm/i915/i915_params.c
index 4a90963..81c2918 100644
--- a/drivers/gpu/drm/i915/i915_params.c
+++ b/drivers/gpu/drm/i915/i915_params.c
@@ -74,7 +74,8 @@ struct i915_params i915_modparams __read_mostly = {
 	.enable_dpcd_backlight = false,
 	.enable_gvt = false,
 	.enable_pvmmio = PVMMIO_ELSP_SUBMIT | PVMMIO_PLANE_UPDATE |
-			 PVMMIO_PLANE_WM_UPDATE | PVMMIO_PPGTT_UPDATE,
+			 PVMMIO_PLANE_WM_UPDATE | PVMMIO_PPGTT_UPDATE |
+			 PVMMIO_GGTT_UPDATE,
 	.enable_gvt_oos = 1,
 	.enable_conformance_check = true,
 	.disable_gvt_fw_loading = true,
-- 
2.7.4

